paimon-common/src/main/java/org/apache/paimon/CoreOptions.java

```diff
@@ -1893,6 +1904,10 @@ public String recordLevelTimeField() {
         return options.get(RECORD_LEVEL_TIME_FIELD);
     }
 
+    public boolean changelogProducerLookupWait() {
```

Comment:
Does it have to be used with lookup?

Reply:
good question! I will modify this method to waitCompaction, and add lookup judgement.

Reply:
back to name `prepareCommitWaitCompaction`.